server/templates/pages/src/index.html

```diff
@@ -32,7 +32,7 @@
           <!-- endbuild -->
         <![endif]-->
     </head>
-    <body>
+    <body{{#if flowId}} data-flow-id="{{flowId}}" data-flow-time="{{flowTime}}"{{/if}}>
```

Comment:
I cannot get this to work, the flowId is always undefined, does it work for you?